### ISSUE TITLE
Training loss becoming NaN

### ISSUE THREAD

**Issue by hillaric** [CoinCheung/BiSeNet#91]
I use BiseNetv2 to train my own dataset, but the loss becoming NaN if I finish the evaluation, just like this picture.
![image](https://user-images.githubusercontent.com/50513602/97100603-06eaec80-16d0-11eb-8a4c-739772e02fc4.png)

**Comment by hillaric** [comment]
I need your help, my old iron! @CoinCheung

**Comment by CoinCheung** [comment]
Hi, 
Sorry for being so late, I am busy dealing with our "money dad"... What sort of dataset are you using ? How many classes are there in your dataset ?

**Comment by hillaric** [comment]
thanks for your reply. Who is money dad? I use 21 classes, there are the objects in the room, like cup, bowl.

**Comment by hillaric** [comment]
In my opinion, it is the problem of evaluation. The loss is normal if I continue to train, but I don't know the current accuracy.

**Comment by CoinCheung** [comment]
So you are not using my original code. What modifications have you made, and can your modification work with cityscapes ?

**Comment by hillaric** [comment]
Yes, I change the cityscapes_dict in the cityscape.py, the number of the classes and the epoch to save the model
recent:///9889acfd07df125e1a3b5c285f9295d4

**Comment by hillaric** [comment]
In the train.py 
file:///home/zhou/Pictures/2020-10-27%2019-21-50%E5%B1%8F%E5%B9%95%E6%88%AA%E5%9B%BE.png

**Comment by hillaric** [comment]
In the train.py.
![2020-10-27 19-21-50屏幕截图](https://user-images.githubusercontent.com/50513602/97295669-920fe200-188a-11eb-92df-6c080aa6d0a5.png)

**Comment by CoinCheung** [comment]
Maybe you can add a line of `net.train()` after the blocked you added.

**Comment by hillaric** [comment]
Yes, it works! Could you give some suggestions about the indoor segmentation, thanks for your help!

**Comment by CoinCheung** [comment]
I am not familiar with indoor segmentation, but it is always good idea to use more training data. And maybe you need to be more careful about picking hyper-parameters.

**Comment by hillaric** [comment]
thanks, you are right!
